lib/matplotlib/patches.py

```diff
@@ -771,15 +788,18 @@ def set_bounds(self, *args):
             l, b, w, h = args[0]
         else:
             l, b, w, h = args
-        self._x = l
-        self._y = b
+        self._x0 = l
+        self._y0 = b
         self._width = w
         self._height = h
+        self._update_x1()
+        self._update_y1()
         self.stale = True
 
     def get_bbox(self):
-        return transforms.Bbox.from_bounds(self._x, self._y,
-                                           self._width, self._height)
+        x0, y0, x1, y1 = self._convert_units()
```

Comment:
Why this call to `convert_units`?  Am I missing a side effect or was this meant to be passed to the `from_extents` call?

Reply:
The `_x0`, `_y0` etc. coords are now all stored in the class with units attached, so this strips the units because I don't think `from_extents` takes units.

Reply:
but the next line passes in `self._x0` etc not `x0`.

Reply:
Ah, I see. PR incoming